src/cloudant/scheduler.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+# Copyright (C) 2018 IBM Corp. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+TODO
+"""
+
+class Scheduler(object):
```

Comment:
Are we ignoring these other scheduler endpoints?
 - GET `/_scheduler/docs/{replicator_db}`
 - GET `/_scheduler/docs/{replicator_db}/{docid}`
They seem to just to doc fetches so I guess there's not much value add there.

Reply:
Good point. On java-cloudant we just did `/_scheduler/docs/_replicator/{docid}` on the assumption that having a different replication DB is so niche that most people won't need it, What do you think?

Reply:
The `replicator.py` uses `_replicator` so I think this is fine without the custom versions.